src/Abstractions/NexusMods.Abstractions.Loadouts.Synchronizers/ALoadoutSynchronizer.cs

```diff
@@ -495,12 +495,13 @@ await _fileStore.ExtractFiles(toExtract.Select(item =>
                     throw new InvalidOperationException("File found in tree processing is not a loadout file, this should not happen (until generated files are implemented)");
                 }
 
+                var writeTimeUtc = register.GetResolvedPath(entry.Path).FileInfo.LastWriteTimeUtc;
```

Comment:
```suggestion
                var writeTimeUtc = new DateTimeOffset(register.GetResolvedPath(entry.Path).FileInfo.LastWriteTimeUtc);
```

Functionality the same as before except it's now explicit that we're using `DateTimeOffset` instead of using the implicit conversion from `DateTime` to `DateTimeOffset`.

We've had funky exceptions going from `DateTime` to `DateTimeOffset` before, especially when dealing with default or epoch values, this just makes sure any exceptions related to the conversion happen in this line.